tests/test.spec.js

```diff
@@ -0,0 +1,37 @@
+const { test, expect } = require('playwright/test')
+
+const login = 'standard_user';
+const password = 'secret_sauce';
+
+test.describe('Playwright Home Task 1', () => {
+
+  test.beforeEach(async ({ page }) => {
+
+    await page.goto('/');
```

Comment:
Since we have `https://www.saucedemo.com/`, this navigation seens to have target `https://www.saucedemo.com//`. Lets update baseUrl or this goto parameter